Description: Convert to build non-thin libs
Convert to build non-thin libs for distribution

--- a/pdfium.gyp
+++ b/pdfium.gyp
@@ -36,7 +36,7 @@
     {
       'target_name': 'pdfium',
       'type': 'static_library',
-
+      'standalone_static_library': 1,
       'dependencies': [
         'third_party/third_party.gyp:bigint',
         'third_party/third_party.gyp:freetype',
@@ -129,6 +129,7 @@
     {
       'target_name': 'fdrm',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'ldflags': [ '-L<(PRODUCT_DIR)',],
       'sources': [
         'core/include/fdrm/fx_crypt.h',
@@ -140,6 +141,7 @@
     {
       'target_name': 'fpdfdoc',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'ldflags': [ '-L<(PRODUCT_DIR)',],
       'sources': [
         'core/include/fpdfdoc/fpdf_ap.h',
@@ -169,6 +171,7 @@
     {
       'target_name': 'fpdfapi',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'ldflags': [ '-L<(PRODUCT_DIR)',],
       'sources': [
         'core/include/fpdfapi/fpdfapi.h',
@@ -291,6 +294,7 @@
     {
       'target_name': 'fpdftext',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'ldflags': [ '-L<(PRODUCT_DIR)',],
       'sources': [
         'core/include/fpdftext/fpdf_text.h',
@@ -306,6 +310,7 @@
     {
       'target_name': 'fxcodec',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'include_dirs': [
       ],
       'ldflags': [ '-L<(PRODUCT_DIR)',],
@@ -488,6 +493,7 @@
     {
       'target_name': 'fxcrt',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'ldflags': [ '-L<(PRODUCT_DIR)',],
       'sources': [
         'core/include/fxcrt/fx_arb.h',
@@ -533,6 +539,7 @@
     {
       'target_name': 'fxge',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'ldflags': [ '-L<(PRODUCT_DIR)',],
       'sources': [
         'core/include/fxge/fpf.h',
@@ -645,6 +652,7 @@
     {
       'target_name': 'fxedit',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'ldflags': [ '-L<(PRODUCT_DIR)',],
       'sources': [
         'fpdfsdk/include/fxedit/fx_edit.h',
@@ -661,6 +669,7 @@
     {
       'target_name': 'pdfwindow',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'ldflags': [ '-L<(PRODUCT_DIR)',],
       'sources': [
         'fpdfsdk/include/pdfwindow/IPDFWindow.h',
@@ -704,6 +713,7 @@
     {
       'target_name': 'javascript',
       'type': 'static_library',
+      'standalone_static_library': 1,
         'include_dirs': [
           '<(DEPTH)/v8',
           '<(DEPTH)/v8/include',
@@ -764,6 +774,7 @@
     {
       'target_name': 'jsapi',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'dependencies': [
         '<(DEPTH)/v8/tools/gyp/v8.gyp:v8',
       ],
@@ -783,6 +794,7 @@
     {
       'target_name': 'formfiller',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'ldflags': [ '-L<(PRODUCT_DIR)',],
       'sources': [
         'fpdfsdk/include/formfiller/FFL_CBA_Fontmap.h',
